src/server/create.rs

```diff
@@ -216,6 +216,14 @@ pub fn create(options: &Create) -> anyhow::Result<()> {
     let (distr, meth_name, method) = find_distribution(
         &*current_os, &avail_methods,
         &version_query, &options.method)?;
+    let credentials_path = credentials::path(&options.name)?;
+    if credentials_path.exists() && !options.overwrite {
+        anyhow::bail!("Credential file {} already exists. \
+            This may mean that instance is already initialized. \
```

Comment:
What else can it mean?

Reply:
Probably we could say "This means the instance is already initialized, or the name is taken as a link to a remote instance"?